### PR TITLE
Bugfix: Configuration for 2019 Geometry

### PR DESCRIPTION
**Pull request by ianna**
* Use Extended 2019 from DB

### CODE REVIEW

**Comment by ianna** [comment]
hold

**Comment by ianna** [comment]
@fabiocos - This PR will need a new GT

**Comment by ianna** [comment]
unhold

**Comment by ianna** [comment]
please test

**Comment by fabiocos** [comment]
please test

**Comment by fabiocos** [comment]
@ianna this PR looks to include also #24509 that is unrelated from the purpose of this one, am I correct?

**Comment by ianna** [comment]
@fabiocos - yes, the payloads are produced using it.

**Comment by ianna** [comment]
@kpedro88 - the errors in the workflows are due to adding a run3 modifier to include a GEM geometry producer from DB. I think, I need help with that. Thanks.
```
----- Begin Fatal Exception 02-Oct-2018 10:07:59 CEST-----------------------
An exception of category 'NoProxyException' occurred while
   [0] Processing  stream begin Run run: 1 stream: 0
   [1] Calling method for module GEMPadDigiClusterProducer/'simMuonGEMPadDigiClusters'
Exception Message:
No data of type "GEMGeometry" with label "" in record "MuonGeometryRecord"
 Please add an ESSource or ESProducer to your job which can deliver this data.
----- End Fatal Exception -------------------------------------------------

```

**Comment by ianna** [comment]
-1

@fabiocos - please, use #24782 instead of this one for the GT update.

File: Configuration/Geometry/python/GeometryRecoDB_cff.py [hunk 1]
```diff
@@ -32,3 +32,6 @@ def _loadGeometryESProducers( theProcess ) :
 
 from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
 modifyGeometryConfiguration_cff_ = run2_GEM_2017.makeProcessModifier( _loadGeometryESProducers )
+
+from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
+modifyGeometryConfiguration_cff_ = run3_GEM.makeProcessModifier( _loadGeometryESProducers )
```
Comment (kpedro88) [inline Configuration/Geometry/python/GeometryRecoDB_cff.py]: @ianna each modifier object needs to have a unique name. I changed the names to `modifyGeometryConfigurationRun2_cff_` and `modifyGeometryConfigurationRun3_cff_`, and workflow 10024.0 step2 runs to completion.

Comment (ianna) [reply Configuration/Geometry/python/GeometryRecoDB_cff.py]: @kpedro88 - Thanks! I'll open a separate PR for it.

**Pull request closed without merging**